sensor/kubernetes/eventpipeline/pipeline.go

```diff
@@ -17,7 +17,16 @@ import (
 	"github.com/stackrox/rox/sensor/kubernetes/listener/resources"
 )
 
-// New instantiates the eventPipeline component
+// New instantiates the eventPipeline component. Sensors pipeline is responsible for the entire lifecycle of a Kubernetes
+// event. From receiving it from the listeners, converting it to storage.Deployment, finding related resources to be
+// reprocessed, generating alerts, and finally sending messages to Central.
```

Comment:
But this pipeline can also send messages to the Detector?
In the docs sometimes you are referring to both, or only Central or the Detector. What is the difference between the two?

Reply:
This was actually inacurate. Rewrote for clarity:
> New instantiates the eventPipeline component. Sensors pipeline is responsible for the entire lifecycle of a Kubernetes event. From receiving it from the listeners, converting it to storage.Deployment, finding related resources to be reprocessed, and sending events directly to Central or to the detector for processing.